applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                                <field name="voltage" type="NX_NUMBER"/>
+                                <field name="extraction_voltage" type="NX_NUMBER" optional="true"/>
+                                <field name="emission_current" type="NX_NUMBER" optional="true"/>
+                                <field name="filament_current" type="NX_NUMBER" optional="true"/>
+                            </group>
+                            <group type="NXlens_em" minOccurs="0" maxOccurs="unbounded">
+                                <doc>
+                                    Instances should use lens as a name prefix.
+                                </doc>
+                                <field name="power_setting" type="NX_CHAR_OR_NUMBER"/>
+                            </group>
+                            <group type="NXaperture" minOccurs="0" maxOccurs="unbounded">
+                                <doc>
+                                    Instances should use aperture as a name prefix.
+                                </doc>
+                                <field name="setting" type="NX_CHAR_OR_NUMBER">
+                                    <doc>
+                                        Descriptor for the aperture setting when the exact technical details
+                                        are unknown or not directly controllable as the control software of
+                                        the microscope does not enable or was not configured to display these
+                                        values for users.
+                                    </doc>
+                                </field>
+                            </group>
+                            <group type="NXmonochromator" minOccurs="0" maxOccurs="unbounded">
+                                <doc>
+                                    Instances should use monochromator as a name prefix.
+                                </doc>
+                                <field name="applied" type="NX_BOOLEAN"/>
+                                <field name="dispersion" type="NX_NUMBER" recommended="true"/>
+                                <field name="voltage" type="NX_NUMBER" recommended="true"/>
+                            </group>
+                            <group name="corrector_cs" type="NXcorrector_cs" minOccurs="0" maxOccurs="1">
+                                <field name="applied" type="NX_BOOLEAN" recommended="true"/>
+                                <group name="TABLEAU" type="NXprocess" nameType="any" minOccurs="1" maxOccurs="unbounded">
+                                    <!--model(NX_CHAR):-->
+                                    <!--ceos-->
+                                    <doc>
+                                        Instances should use tableau as a name prefix.
+                                    </doc>
+                                    <group name="c_1" type="NXaberration" optional="true">
```

Comment:
What are all these c_1, a_1, etc. fields?

Reply:
Individual named aberration coefficients, all "specified", following the naming convention of the electron microscopy community.

Reply:
Comment can be resolved as it was just to communicate purpose